www/js/con-author.js

```diff
@@ -211,12 +354,23 @@ define([
 			*/
 		},
 
-		handleSetStudentQtyNode: function(checked){
+		handleSetStudentNode: function(checked){
 			console.log("********************* in handle set student quantity node", checked);
-		},
-
-		handleSetStudentEqNode: function(checked){
-			console.log("********************* in handle set student equation node", checked);
+			if(checked){
+				style.set('modelSelectorContainer', 'display', 'block');
+				var studentNode = this._model.student.getNodeIDFor(this.currentID);
+				if(studentNode == null){
+					//to do : to write addStudentNode which has to add a quantity node, pushing this it backlog 
+					//this.addStudentNode(this.currentID);
```

Comment:
Oh I see, made changes. Though I don't get this highlighted in sublime, will check it further but made the changes